### PR TITLE
Card Integration Guide

### PR DESCRIPTION
**Pull request by sshropshire**
### Summary of changes

 - This PR updates integration docs for the `Card` module

 ### Checklist

 ~- [ ] Added a changelog entry~

### Authors

- @sshropshire

### CODE REVIEW

File: docs/Card/README.md [hunk 1]
```diff
@@ -118,24 +109,37 @@ Attach the card and the order ID from [step 3](#3-create-an-order) to a `CardReq
...
+
+    func approveOrder(cardRequest: CardRequest) {
+        Task {
+            do {
+                let result = cardClient.approveOrder(request: cardRequest, context: self)
+                // order was successfully approved and is ready to be captured/authorized (see step 6)
+            } catch {
+                // handle the error by accessing `error.localizedDescription`
+            }
+        }
```
Comment (jcnoriega) [inline docs/Card/README.md]: this is no longer the case, because we are now opening the browser, we switched to the `delegate` pattern.

Comment (jcnoriega) [reply docs/Card/README.md]: but we can update this once the other PR is merged

Comment (sshropshire) [reply docs/Card/README.md]: Right yeah I can update this.

**Review by jaxdesmarais — APPROVED** [review]
⛵

**Review by jcnoriega — APPROVED** [review]

**Pull request merged**